apps/connect/src/components/editors.tsx

```diff
@@ -187,32 +182,6 @@ export const DocumentEditor: React.FC<Props> = (props) => {
 
   return (
     <div className="relative h-full" id="document-editor-context">
-      {!isExternalControlsEnabled && !revisionHistoryVisible && (
-        <DocumentToolbar
-          onClose={onClose}
-          onExport={onExport}
-          onShowRevisionHistory={() => setRevisionHistoryVisible(true)}
-          title={documentName}
-          timelineItems={timelineItems.data}
-          onTimelineItemClick={setSelectedTimelineItem}
-        />
-      )}
-      {!!isExternalControlsEnabled && (
```

Comment:
I wasn't sure if we wanted to remove this completely or not, lmk if this needs to stay so I can undo this change